lib/firefly.ts

```diff
@@ -130,6 +132,17 @@ export default class FireFly extends HttpBase {
     );
   }
 
+  getNamespace(
+    name: string,
+    options?: FireFlyGetOptions,
+  ): Promise<FireFlyNamespaceResponse | undefined> {
+    return this.getOne<FireFlyNamespaceResponse>(`/namespaces/${name}`, undefined, options, true);
+  }
+
+  getNamespaces(options?: FireFlyGetOptions): Promise<FireFlyNamespacesResponse[]> {
+    return this.getMany<FireFlyNamespacesResponse[]>('/namespaces', undefined, options, true);
```

Comment:
I think you want either `FireFlyNamespacesResponse` or `FireFlyNamespaceResponse[]`. We usually prefer the latter, unless there is a major difference in the fields returned by "get many" to warrant two separate types.

I think the type you've defined is a doubly nested array, which won't be correct.

Reply:
@awrichar Thanks for spotting it. I'll change it to use `FireFlyNamespaceResponse[]` Weirdly, running the code as it is did return the expected format when I tested it. I guess the type-matching is not that strict?

Reply:
@awrichar updated PR raised here: https://github.com/hyperledger/firefly-sdk-nodejs/pull/57